src/cargo/util/semver_eval_ext.rs

```diff
@@ -75,7 +75,7 @@ fn matches_exact(cmp: &Comparator, ver: &Version) -> bool {
 }
 
 // See https://github.com/dtolnay/semver/blob/69efd3cc770ead273a06ad1788477b3092996d29/src/eval.rs#L64-L88
-fn matches_greater(cmp: &Comparator, ver: &Version) -> bool {
+pub(crate) fn matches_greater(cmp: &Comparator, ver: &Version) -> bool {
```

Comment:
Unsure if this commit is worth it.  In theory, `semver_eval_ext` will be going away and we'll have to undo this.

Reply:
Well, this commit is not very important for this PR.  We can decide what to do with it later.